src/formatDate.js

```diff
@@ -50,7 +50,30 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const currentDateOld = date.split(fromFormat[3]);
+  const res = [];
+  const dateInfo = {};
+
+  for (let i = 0; i < 3; i++) {
+    if (fromFormat[i] === 'YY') {
+      if (currentDateOld[i] < 30) {
+        dateInfo['YYYY'] = `20${currentDateOld[i]}`;
+      } else {
+        dateInfo['YYYY'] = `19${currentDateOld[i]}`;
+      }
```

Comment:
Try to avoid using `if else `statement
I would rather use ternary operator here. It looks clear